boto3_fixtures/sns.py

```diff
@@ -0,0 +1,67 @@
+"""
+Module `sns` implements the setup and tear-down of SNS topics via boto3.
+
+Example:
+    >>> import boto3_fixtures as b3f
+    >>> # ... setup `aws` fixture
+    >>>
+    >>> TOPICS = [
+    >>>     "topic-1",
+    >>>     {
+    >>>         "TopicName": "topic-with-attrs",
+    >>>         "Attributes": {"DisplayName": "Topic With Attributes"},
+    >>>     }
+    >>> ]
+    >>> sns = b3f.contrib.pytest.service_fixture(sns, topics=TOPICS)
+"""
+
+import typing as T
+
+import backoff
+from botocore.exceptions import ClientError
+
+import boto3_fixtures.contrib.boto3
+import boto3_fixtures.utils as utils
+
```

Comment:
Needs a namedtuple for SNSTopic. Ideally with `name`, `arn`, and `response`. All of the fixtures return similar objects to the interface should stay consistent!

Reply:
The main reason it exists is to make debugging super easy. If you include this fixture on a test, you can examine the resources easily with pdb when something fails.

Reply:
:+1: updated.